src/components/MainSidebar.vue

```diff
@@ -38,12 +38,14 @@ onMounted(() => {
   >
     <div class="flex flex-col h-full overflow-scroll menu-tabs">
       <div class="min-h-[78px] h-[78px] flex items-center justify-center">
-        <Icon
-          @click="$router.push({ name: 'home' })"
-          size="36"
-          name="snapshot"
-          class="text-snapshot cursor-pointer"
-        />
+        <router-link :to="{ name: 'home' }">
```

Comment:
Why do we need to link to route twice here?

Reply:
Ah forgot to remove it from icon.